docs/guides/component-testing/getting-started.mdx

```diff
@@ -0,0 +1,622 @@
+---
+title: Cypress Component Testing
+sidebar_position: 20
+sidebar_label: Getting Started
+---
+
+Getting started with Component Testing is super simple. Follow the guide below for a step by step guide on how to configure your project and quickly begin writing your first tests in just a few seconds.
+
+## Getting Started
+
+<DocsVideo
+  src="/img/snippets/getting-started-with-ct.mp4"
+  title="Getting Started With Component Testing"
+/>
+
+### Install Cypress
+
+To begin we need to install Cypress in your project if you have not already done so.
+
+```bash
+npm install cypress -D
+```
+
+### Open Cypress:
+
+Once Cypress has been installed you can launch Cypress using the following command:
+
+```bash
+npx cypress open
```

Comment:
We can save a step if people open with `--component`, I wonder if that is worth mentioning.